docs/INSTALL.rst

```diff
@@ -99,27 +104,79 @@ setups, see ``docs/network-drivers.md``.
 Logging level and directory can be set in the ``[general]`` section. For more
 information view ``docs/logging.md``.
 
-The file should be placed at ``/etc/haas.cfg``; The ``haas.wsgi``
-script, described below, requires this. Awkwardly, the ``haas``
-command line tool expects the file to be present in its current
-working directory. This will be fixed in the next release, but for
-now, put the file in ``/etc`` and create a symlink to it in the
-HaaS user's home directory::
 
-  sudo ln -s /etc/haas.cfg /var/lib/haas/
+``haas.cfg`` file contains sensitive administrative information and should not be exposed to clients or 
+end users. Therefore, after placing the file at ``/etc/haas.cfg`` its 
+permissions should be set to read-only and ownership set to ``${HAAS_USER}``
```

Comment:
we could I guess, but this is how we have been doing so far. I do not see the need to relax it.